devtools-ci-index.yaml

```diff
@@ -97,7 +97,28 @@
             # testing out the cico client
             set +e
             export CICO_API_KEY=$(cat ~/duffy.key )
-            read CICO_hostname CICO_ssid <<< $(cico node get -f value -c ip_address -c comment)
+            # get node
```

Comment:
@hectorj2f the intention is to try 5 times, at 60 second interval's and then exit if no node can be reserved.